tools/shared.py

```diff
@@ -2979,8 +2979,20 @@ def delebify(buf, offset):
 
   @staticmethod
   def add_emscripten_metadata(js_file, wasm_file):
-    mem_size = Settings.STATIC_BUMP
+    WASM_PAGE_SIZE = 65536
```

Comment:
so STATIC_BUMP is no longer needed?

Reply:
STATIC_BUMP was being used to compute `DYNAMICTOP_PTR` and `tempDoublePtr` but since those are now not derived from `STATIC_BUMP` it's no longer necessary to store that.